Example/Backpack/ViewControllers/BPKRootListTableViewController.swift

```diff
@@ -20,22 +20,75 @@ import UIKit
 import Backpack
 
 class BPKRootListTableViewController: UITableViewController {
```

Comment:
We should probably split this into two classes, one reponsible for app structure and one responsible for the settings